lib/mix/lib/mix/local/installer.ex

```diff
@@ -113,17 +117,11 @@ defmodule Mix.Local.Installer do
           Mix.raise message
 
         {kind, message} when kind in [:remote, :checksum] ->
-          Mix.raise """
-          #{message}
-
-          Could not fetch #{name} at:
-
-              #{src}
-
-          Please download the #{name} above manually to your current directory and run:
-
-              mix #{name}.install ./#{src_basename}
-          """
```

Comment:
@josevalim: do you want to do anything about this one that is broken in different lines?